module/x/gravity/abci.go

```diff
@@ -128,6 +141,10 @@ func eventVoteRecordTally(ctx sdk.Context, k keeper.Keeper) {
 		// They are ordered by when the first attestation at the event nonce was received.
 		// This order is not important.
 		for _, att := range attmap[nonce] {
```

Comment:
is the iteration deterministic?

Reply:
I think its ordered by the attestation hash. but seems it doesnt matter